apps/src/templates/SchoolNameInput.jsx

```diff
@@ -1,28 +1,30 @@
 import PropTypes from 'prop-types';
-import React, {useState} from 'react';
+import React from 'react';
 
 import TextField from '@cdo/apps/componentLibrary/textField/TextField';
 import i18n from '@cdo/locale';
 
-import {SCHOOL_NAME_SESSION_KEY} from '../signUpFlow/signUpFlowConstants';
-
-export default function SchoolNameInput({fieldNames}) {
-  const detectedName = sessionStorage.getItem(SCHOOL_NAME_SESSION_KEY) || '';
-  const [schoolName, setSchoolName] = useState(detectedName);
+export default function SchoolNameInput({
+  fieldNames,
+  schoolName,
+  setSchoolName,
+}) {
+  const handleSchoolNameChange = name => {
+    setSchoolName(name);
```

Comment:
Same question for this (if this is for readability/consistency that's a-okay with me, just wanted to check if this was adding any functionality or not!)

Reply:
It's mostly just a convention I've gotten used to. definitely could be in the onChange event, but I like seeing all the handler functions in the body of the functional component rather than the returned jsx. Mostly because I don't want any handler logic down there (but these don't have any logic anymore 😁 )